src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -217,6 +329,11 @@ internal void SendWorkerInitRequest(GrpcEvent startEvent)
                 initRequest.Capabilities.Add(RpcWorkerConstants.FunctionDataCache, "true");
             }
 
+            // advertise that we support multiple streams, and hint at a number; with this flag, we allow
+            // clients to connect multiple back-hauls *with the same workerid*, and rely on the internal
+            // plumbing to make sure we don't process everything N times
+            initRequest.Capabilities.Add(RpcWorkerConstants.MultiStream, "10"); // TODO: make this configurable
```

Comment:
Can we make the number ("10") a readonly constant for now?

Is the TODO for this PR or future work? If future work, can we create a github issue for this and mention it here?